plugins/boltz/README.md

```diff
@@ -1,21 +1,83 @@
+go on and create another swap to bring liquidity in line with desired settings. This setting is for one rebalancing event that can cause multiple swaps.
+
+If the workflow action is triggered again (e.g., liquidity goes above target and then again below) then another rebalancing is attempted and it again has the full 5% "allowance".
+
+Global flags are used in a meanigful way, for instance if `--network testnet` then `--boltzurl` is changed to point to testnet boltz URL.
+
+### CLI mode
+
+```
+NAME:
+   bolt-agent boltz - interact with boltz plugin
+
+USAGE:
+   bolt-agent boltz command [command options] [arguments...]
+
+COMMANDS:
+   submarineswap         invoke submarine swap aka swap-in (on-chain -> off-chain)
+   reversesubmarineswap  invoke reverse submarine swap aka swap-out (off-chain -> on-chain)
+   generaterefund        generate refund file to be used directly on boltz exchange GUI
+   dumpmnemonic          print master secret as mnemonic phrase (dangerous)
+   setmnemonic           set a new secret from mnemonic phrase (dangerous)
+
+OPTIONS:
+   --help, -h  show help
+```
+
+Plugin can be executed in standalone CLI mode too which enables you to do submarine or reverse submarine swaps on your node immediately.
+
+You can manually invoke a swap like this:
+
+```
+bolt-agent boltz submarineswap --sats 100000
+```
+
+ID is reported so you can continue doing a swap also if application died in the middle.
+Just add `--id -1337` where -1337 is the ID reported and swap will be resumed from where it stopped before.
+
+`generaterefund` generates a JSON file that can be used on Boltz GUI
+
+#### Mnemonic
+
+`dumpmnemonic` and `setmnemonic` should not be used, but they allow you to change entropy (also stored in database). All secrets (like keys and preimages are generated from it, so you should never write this down anywere!).
+The command can be useful if agent died in the middle of a swap and you want to resume the swap using CLI mode on another node (you just `dumpmnemonic` and import entropy on new node using `setmnemonic`).
+Then same ID should result in same secrets and you can for instance use `submarineswap` on second node to finish the swap (or get a refund).
 
 ## Options
 
 ```
-   --boltzurl value           url of boltz api (default: "https://boltz.exchange/api")
+   --boltzurl value           url of boltz api - empty means default - https://boltz.exchange/api or https://testnet.boltz.exchange/api
    --boltzdatabase value      full path to database file (file will be created if it does not exist yet) (default: "/home/user/.bolt/boltz.db")
-   --dumpmnemonic             should we print master secret as mnemonic phrase (dangerous)
-   --setmnemonic value        update saved secret with this key material (dangerous)
+   --boltzreferral value      boltz referral code (default: bolt-observer)
```

Comment:
when did we decide this will be adjustable by users?

Reply:
It is adjustable by users per definition since this code is open source. I thought playing with open cards is better, it's a hidden parameter that people won't need to be bothered with (but if they want to change it they can, instead of forking the code and altering some compile time constant).